### PR TITLE
Acd 10956 update fixction broken

### PR DESCRIPTION
**Pull request by meetshyani-crest**
This PR contains the code for:

- Updating the details for the fiction addon stanzas
- Adding the scenario for tokenizing the samples through the lookup file in fiction addon
- Adding the scenario for providing the wrong timezone in the conf file
- Update in the lookup file rule to tokenize all the tokens in the sample file through the lookup value

### CODE REVIEW

File: tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf [hunk 1]
```diff
@@ -1,3 +1,24 @@
+## Total test cases: 98
+## Passed: 98
```
Comment (bhargav-nariyani-crest) [inline tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf:2]: Please update

File: tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+## Skipped: 4
+
+## Search Time Tests
+## Passed: 27
+## Skipped: 4
+
+## Common Test
+## 1 passed (Find untokenised values in events)
+
+# Covers almost ll the rules supported by pytest-splunk-addon plugin
```
Comment (bhargav-nariyani-crest) [inline tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf:13]: Fix typo

File: tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf (hunk 1 above)
Comment (meetshyani-crest) [reply tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf]: Updated the test cases count

File: tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf (hunk 2 above)
Comment (meetshyani-crest) [reply tests/addons/TA_fiction_indextime/default/pytest-splunk-addon-data.conf]: Fixed.

**Pull request merged**